app/server/models/transfer_usage.py

```diff
@@ -29,3 +30,11 @@ def icon(self, icon):
         if icon not in MATERIAL_COMMUNITY_ICONS:
             raise IconNotSupportedException('Icon {} not supported or found')
         self._icon = icon
+
+    @hybrid_property
+    def name(self):
+        return self._name
+
+    @name.setter
+    def name(self, name):
+        self._name = name.strip().upper()
```

Comment:
what's the reasoning around why we coerce this into uppercase before storing? That means if we use the name anywhere on the UX it's be necessarily uppercase...

If it's for testing for duplicates, we could just do that using a filter with an extra hybrid property such as name_lower?

Reply:
right, to prevent dupes when inserting new ones. I think coercing them to all be uniform when setting isn't so bad, but maybe instead of upper, it should be capitalized?

Reply:
Capitalised would be better, but i'm worried about unnecessarily stripping out information - what happens if a category contained an acronym like "WASH Supplies"? (WAter, Sanitation and Healthcare)

Reply:
Okay, shall I remove the capitalizing from this method? Instead I can add something like this in the setter method: `SELECT * 
FROM public.transfer_usage where lower('food') = lower(_name)` and then see if it return one and throw an error if does if it returns nothing insert into the database. So that way we can ensure the the capitalizing remains the way it's intended by the user and still prevent dupes.

Reply:
We only take the risk that a user really want to add a WASH as a different category then wash this is prevented. But that really seems like an edge case

Reply:
> But that really seems like an edge case

Hahaha yeah agreed lol! It should be possible to code this up without writing raw SQL...